packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -323,8 +322,8 @@ abstract contract MajorityVotingBase is
     /// @return True if the proposal vote is open, false otherwise.
     function _isVoteOpen(Proposal storage proposal_) internal view virtual returns (bool) {
         return
-            getTimestamp64() < proposal_.endDate &&
-            getTimestamp64() >= proposal_.startDate &&
+            getTimestamp64() < proposal_.configuration.endDate &&
```

Comment:
1. `getTimestamp64` is called 2 times. Let's call one time and use its result in these 2 places.. 
2. It would be more logical if we do: 
```
proposal_.configuration.startDate < timestamp &&
proposal_.configuration.endDate >= timestamp
```

Reply:
Amended in aa35c21ba27ce6f8df4580e8f816bea7bc7a9b7d